### PR TITLE
fix: DateTimePicker 'Default time value' Invalid Date error

### LINKED ISSUE #822: DateTimePicker 设定默认的起始与结束时刻后，选择日期，显示Invalid Date
image

### PR DESCRIPTION
**Pull request by wushuxuan**
Close #822 
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by JeremyWuuuuu** [comment]
Seems like the test cases failed, please fix that. Also, the convention is that add `space` after each token/operator.

File: packages/date-picker/__tests__/date-time-picker.spec.ts [hunk 1]
```diff
@@ -267,51 +267,6 @@ describe('Datetime Picker', () => {
...
-    }
-    await nextTick()
-    // both input shows correct value
-    expect((left.dateInput as HTMLInputElement).value).toBe('2000/11/01')
-    expect((left.timeInput as HTMLInputElement).value).toBe('01:01 AM')
-    expect((right.dateInput as HTMLInputElement).value).toBe('2000/12/01')
-    expect((right.timeInput as HTMLInputElement).value).toBe('01:01 AM')
-  })
-
   it('input date', async () => {
```
Comment (JeremyWuuuuu) [inline packages/date-picker/__tests__/date-time-picker.spec.ts:270]: Why are we removing all these testings?

Comment (JeremyWuuuuu) [reply packages/date-picker/__tests__/date-time-picker.spec.ts:270]: If your code didn't pass, maybe there is something wrong with your code.

**Comment by iamkun** [comment]
doc updated here #851

**Review by zazzaz — COMMENTED** [review]
update docs will be a better choice

**Pull request closed without merging**